### ISSUE TITLE
[Feature Request] Backup changed data

### ISSUE THREAD

**Issue by aiolos01** [NeoApplications/Neo-Backup#706]
Titanium backup has the extremely useful feature of backing up changed application data. This is much more important than backing up changed apks because the current version of an app can always be downloaded from the play store or sideloaded but if the most recent data is lost then it's gone for good.

Currently the only way I can see to make sure all recently changed data is backed up is to redo all app backups which is both time (and battery) consuming and stressful for the flash memory. Writing 10+GB is not something you want to do every day or so.

It would be very useful to have the capability of scheduling a "changed data backup: every day. As far as my use case is concerned this is the only thing missing from NB and the reason I keep using TB as well.

**Comment by machiav3lli** [comment]
tldr;
have been discussed in the community multiple times (and I guess there has been couple of such issues) and the outcome was a clear NOT PLAUSIBLE.

Data flow has been radically changed since TiB was written (apps data change way much often than before, also when not running) and file access get complex and limited too (Many restrictions on the different access APIs make the performance unacceptable).
